src/main/java/frc/robot/Constants.java

```diff
@@ -17,19 +17,27 @@
 public final class Constants {
     public static final class JoystickConstants {
         public static final int kDriverControllerPort = 0;
-        public static final int kOperatorControllerPort = 1;
+        public static final int kOperatorControllerPort = 0;
     }
 
 
     public static final class DriveConstants {
-
+        public static final int lefRearMotor = 0;
+        public static final int leftFrontMotor = 0;
+        public static final int rightRearMotor = 0;
+        public static final int rightFrontMotor = 0;
```

Comment:
Constantlar k ile başlamalı. ID'ler birbirinden farklı olmalı 10,11,12,13 gidebilirsin drivetrain için. Hopper değil FunnelConstants olmalı ismi değişecek de bunda da ID'leri 20 den başlayabilirsin ve k ile başlamalı
```suggestion
        public static final int kRightFrontMotor = 0;
```

Reply:
Fixlendi bu kontrol ettim.